compiler/codegen/CForLoop.cpp

```diff
@@ -24,9 +24,35 @@
 #include "build.h"
 #include "codegen.h"
 #include "ForLoop.h"
+#include "driver.h"
 
 #include <algorithm>
 
+#ifdef HAVE_LLVM
+namespace
+{
+  llvm::MDNode* addLoopVectorizationHint(llvm::Instruction* instruction)
```

Comment:
I'd expect this function to be `static` rather than in an anonymous namespace. Arguably a style issue, but making it `static` will fit in with the rest of the compiler. Vs when defining a type, you can't use `static`.